### PR TITLE
Add deploymentconfig to sanity_helpers

### PR DESCRIPTION
**Pull request by prsurve**
Signed-off-by: prsurve <prsurve@redhat.com>

### CODE REVIEW

File: tests/sanity_helpers.py [hunk 1]
```diff
@@ -36,13 +39,14 @@ def health_check(self):
         )
         self.ceph_cluster.cluster_health_check(timeout=60)
 
-    def create_resources(self, pvc_factory, pod_factory, run_io=True):
+    def create_resources(self, pvc_factory, pod_factory, dc_pod_factory, run_io=True):
```
Comment (ebenahar) [inline tests/sanity_helpers.py]: I don't think we need to use pod (not DC) anymore

Comment (prsurve) [reply tests/sanity_helpers.py]: will replace pod_factory with dc_pod_factory in next commit

**Comment by RazTamir** [comment]
I'm missing some background about this change.
Why we abandoned the pod_factory in favor of dc?

**Comment by prsurve** [comment]
@RazTamir I have just replaced pod_factoy with dc_pod_factory, if needed i can revert the last commit and keep both factory_fixtures

**Comment by vasukulkarni** [comment]
@prsurve can you add the verified tag?

**Pull request closed without merging**